docs/api-inference/_toctree.yml

```diff
@@ -0,0 +1,52 @@
+- sections:
+  - local: index
+    title: Serverless Inference API
+  - local: getting_started
+    title: Getting Started
+  - local: supported_models
+    title: Supported Models
+  - local: rate_limits
+    title: Rate Limits
+  title: Getting Started
+- sections:
+  - local: parameters
+    title: Parameters
+  - sections:
+    - local: tasks/audio_classification
```

Comment:
any way to use the same ids as the ones we use on the Hub, i.e. dash-separated?

Reply:
(btw i really like that we'll have a specific URL/page per task now)

Reply:
https://github.com/huggingface/hub-docs/pull/1405